dockerfiles/packaging/Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG DOCKER_REGISTRY=docker.io
```

Comment:
I added this. So that we can override the registry if we'll move the images to internal DD registry for faster/safer builds